core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLBatchTestBase.java

```diff
@@ -166,7 +166,7 @@ protected void cleanPartitions(TimePartitionedFileSet fileSet) throws IOExceptio
       if (locName.endsWith(".parquet")) {
         Path parquetFile = new Path(file.toString());
         AvroParquetReader.Builder<GenericRecord> genericRecordBuilder = AvroParquetReader.builder(parquetFile);
-        ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
+        org.apache.parquet.hadoop.ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
```

Comment:
how come the full name is being used? Can we just remove the parquet.hadoop.ParquetReader import and import this class?